src/Modules/AuthenticationModule/components/Login/Login.tsx

```diff
@@ -1,7 +1,225 @@
-import React from 'react'
+import React, { useEffect } from 'react'
+import Grid from '@mui/material/Grid'; 
+import loginImg from "../../../../assets/Images/login.png";
+import { Box, Button, IconButton, InputAdornment, TextField, Typography } from '@mui/material';
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff';
+import CircularProgress from '@mui/material/CircularProgress';
+import { FieldError, SubmitHandler, useForm } from 'react-hook-form';
+import axios from 'axios';
+import { Link, useNavigate } from 'react-router-dom';
+import { useAuth } from '../../../Context/AuthContext';
+import { useState } from 'react';
+type AuthInputs = {
+  email: string;
+  password: string;
+};
 
 export default function Login() {
+  
+  const navigate = useNavigate();
+  // const { loginData } = useAuth();
+  const { getUserData, baseUrl } = useAuth();
+  const [showPass, setShowPass] = useState(true);
+  const [loading, setLoading] = useState(false);
+  let {register, handleSubmit, formState:{errors}} = useForm<AuthInputs>();
+
+ 
+  const onSubmit: SubmitHandler<AuthInputs> = async (data)=>{
+    try{
+      setLoading(true);
+      const response = await axios.post(
+        `${baseUrl}/admin/users/login`,
+        data,
+      );
+      localStorage.setItem('token',response.data.data.token);
+      console.log('token',response.data.data.token);
+      getUserData();
+      navigate('/DashBoard');
+      console.log(response);
+      
+    } catch (error:any){
+      console.error(error.response?.data?.message || 'An error occurred during login');
+
+    } finally {
+      setLoading(false);
+    }
+  }
+
   return (
-    <div>Login</div>
+    <>
+      <Box 
+        sx={{
+          backgroundColor:'purpl',
+          display:'flex',
+          flexDirection:'column',
+          height:'100vh',
+          justifyContent:'center',
+          gap:{xs:3,sm:5}
+        }}>
+        <Box sx={{ backgroundColor:'blac', paddingTop:4, textAlign:'center',display:{md:'none'}}}>
+              <Typography  sx={{ fontSize:{xs:26, sm:30},fontWeight:500 ,display: 'inline',color:'#3252df' }}>
+                Stay
+              </Typography>
+              <Typography  sx={{ fontSize:{xs:26, sm:30},fontWeight:500 ,display: 'inline',color:'#152c5b' }}>
+                cation.
+              </Typography>
+        </Box>
+        
+        <Grid container 
+          sx={{
+            backgroundColor:'gra',
+            padding:{xs:1, sm:2},
+            justifyContent:{xs:'center' },
+            alignItems:{sm:'center',md:'start'},
+            gap:{md:0,lg:1}
+            
+              
+              }}>
+           <Grid item xs={12} sm={10} md={6} lg={5} sx={{display:'flex',flexDirection:'column',paddingTop:{md:1},paddingInline:{md:4,lg:0}, gap:{md:0,xl:4}, backgroundColor:'greenYello'}}>
```

Comment:
We need to use a formatter like prettier to have consisten writing styles in our application.